common.sh

```diff
@@ -77,8 +77,9 @@ if [ -z "$KNI_INSTALL_FROM_GIT" ]; then
 fi
 
 if env | grep -q "_LOCAL_IMAGE=" ; then
+    export LOCAL_REGISTRY_ADDRESS = ${LOCAL_REGISTRY_ADDRESS:-"192.168.111.1:5000"}
```

Comment:
IMHO `LOCAL_REGISTRY_ADDRESS` shouldn't be exported within condition. 
Being exported in outer block would allow images' mirroring(pr #850) not rely on presence of `*_LOCAL_IMAGE` variables, wdyt?

Reply:
Yeah agreed, and I think the changes in 04_setup_ironic.sh will break without this unless *_LOCAL_IMAGE is specified

Reply:
Done